### PR TITLE
Updated and reformatted protocol list

### PR DESCRIPTION
**Pull request by murrayrm**
Pasquale Stano has refined `protocol-list.md` file in his forked copy, adding some minimal commenting and structure. Maybe this can be new starting point for further editing and keeping up-to-date.

(Pull request created by @murrayrm)

### CODE REVIEW

File: protocol-list.md [hunk 1]
```diff
@@ -1,60 +1,83 @@
...
-
-This is a [protocol](https://www.nature.com/protocolexchange/protocols/3815#/related-articles) for the encapsulation of the TX-TL cell-free protein expression system in the liposomes based on the paper by Miyazaki, M, Chiba, M, Ishiwata, S. Preparation of cell-sized water-in-oil droplets for *in vitro* reconstitution of biological processes in cellular compartments. (2015) Nature. DOI: 10.1038/protex.2015.029
-The modified version can be found at [OpenWetWare](https://openwetware.org/wiki/Preparation_of_cell-sized_water-in-oil_droplets) The protocol was used by Malin Jonsson, Miroslav Gasparek (summer 2017 interns), and Zoila Jurado (PhD student at Caltech) to express the GFP in the liposomes in the summer of 2017 in the Murray Lab at Caltech.
+A modified version of this protocol can be found at [OpenWetWare](https://openwetware.org/wiki/Preparation_of_cell-sized_water-in-oil_droplets) The protocol was used by Malin Jonsson, Miroslav Gasparek (summer 2017 interns), and Zoila Jurado (PhD student at Caltech) to express the GFP in the liposomes in the summer of 2017 in the Murray Lab at Caltech.
 
 Aaron Engelhart (University of Minnesota)
```
Comment (murrayrm) [inline protocol-list.md]: We should remove this information from the protocol list.  It was put here when we were testing out GitHub at the BuildACell workshop.

**Comment by murrayrm** [comment]
Can we add some of the other protocols that were discussed at the BuildACell workshop?  The are available in a [Google Doc](https://docs.google.com/document/d/159O3_5vSHeArbi5eDmAdTi5w00SJC8QkFZ8o_Yfyxzo).

**Comment by murrayrm** [comment]
We should also pull out the list of names that were at the bottom of the file.  Those were just there because we were testing out the GitHub pull request process.

**Comment by p-stano** [comment]
I start to see the big potentiality of GitHub, thanks you all to let me know this. If I go in the tab "File changes", then "Review Changes", what is best to propose very minor changes, a "comment" review or the "request changes" option? 
And, in general, what 'tool' would you recommend to correct minor issues?

**Comment by murrayrm** [comment]
A comment is a good way to suggest a change that you believe is useful though not required.  Note that this only works when there is a pull request that is under consideration (like this one).  You can also comment on code that is in a commit, but nobody is likely to be looking at it => nothing will happen.

In general, for correcting the issues you can either suggest the fix in your comment or you could issue your own pull request by forking a copy of the repository, making changes to your local copy, then requesting those changes be "pulled" into the repository of the person making the original pull request.  It would then be up to that person to accept your changes, at which point they would show up in the main pull request.

Finally, note that this particular pull request is actually coming from *you* (p-stano:master) and so in this case you can make a change in your version of the code (back on your account) and that will show up here.

**Pull request merged**